ui/app/mainui/activitycenter/views/ActivityNotificationNewKeypairFromPairedDevice.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+import utils 1.0
+
+ActivityNotificationBase {
+    id: root
+
+    ctaComponent: StatusLinkText {
+        text: qsTr("View keypair import options")
+        color: Theme.palette.primaryColor1
+        font.pixelSize: Theme.primaryTextFontSize
+        font.weight: Font.Normal
+        onClicked: {
+            Global.changeAppSectionBySectionType(Constants.appSection.profile, Constants.settingsSubsection.wallet)
+            root.closeActivityCenter()
+        }
+    }
+
+    bodyComponent: RowLayout {
+        implicitWidth: parent.width
+
+        spacing: 8
+
+        StatusSmartIdenticon {
+            Layout.preferredWidth: 40
+            Layout.preferredHeight: 40
+            Layout.alignment: Qt.AlignTop
+            Layout.leftMargin: Style.current.padding
+            Layout.topMargin: 2
+
+            asset {
+                width: 24
+                height: 24
+                name: "wallet"
+                color: Theme.palette.primaryColor1
+                bgWidth: 40
```

Comment:
bgWidth > width ?

Reply:
@Khushboo-dev-cpp `width` refers to the icon's width and `bgWidth` to the background's width.
![__3](https://github.com/status-im/status-desktop/assets/86303051/21f0d6bf-9b53-4757-8cc7-b544117fc1a1)